domain/src/test/java/org/oppia/android/domain/classify/rules/fractioninput/FractionInputIsEquivalentToAndInSimplestFormRuleClassifierProviderTest.kt

```diff
@@ -24,15 +24,13 @@ class FractionInputIsEquivalentToAndInSimplestFormRuleClassifierProviderTest {
 
   private val WHOLE_NUMBER_VALUE_TEST_5 =
     InteractionObjectTestBuilder.createWholeNumber(
-      isNegative = false,
       value = 5
     )
   private val WHOLE_NUMBER_VALUE_TEST_2 =
     InteractionObjectTestBuilder.createWholeNumber(
-      isNegative = false,
       value = 2
     )
-  private val NEGATIVE_WHOLE_NUMBER_VALUE_TEST_2 =
+  private val NEGATIVE_INTEGER_VALUE_TEST_2 =
     InteractionObjectTestBuilder.createWholeNumber(
```

Comment:
we are using `createWholeNumber` function to create a negative integer.
which is mathematically incorrect
@anandwana001 @coder2699